iroha/src/block_sync.rs

```diff
@@ -263,19 +263,16 @@ pub mod message {
 
 /// This module contains all configuration related logic.
 pub mod config {
-    use std::env;
+    use iroha_config::derive::Configurable;
+    use serde::{Deserialize, Serialize};
 
-    use iroha_error::{Result, WrapErr};
-    use serde::Deserialize;
-
-    const BATCH_SIZE: &str = "BLOCK_SYNC_BATCH_SIZE";
-    const GOSSIP_PERIOD_MS: &str = "BLOCK_SYNC_GOSSIP_PERIOD_MS";
     const DEFAULT_BATCH_SIZE: u64 = 4;
     const DEFAULT_GOSSIP_PERIOD_MS: u64 = 10000;
 
     /// Configuration for `BlockSynchronizer`.
-    #[derive(Copy, Clone, Deserialize, Debug)]
+    #[derive(Copy, Clone, Deserialize, Serialize, Debug, Configurable)]
```

Comment:
Why do we need `Serialize` for configuration?

Reply:
Because we need to expose our value to readers via `serde_json::Value`

Reply:
Egor Ivkov, [02.04.21 18:42]
[In reply to Ivan Rybin]
Maybe I missed, where do we expose the value to readers in str format

Egor Ivkov, [02.04.21 18:43]
Is it in maintenance endpoint?

Ivan Rybin, [02.04.21 18:43]
[In reply to Egor Ivkov]
In trait. In order to implement get method which returns Json value we need to serialize our config

Ivan Rybin, [02.04.21 18:44]
In future it will be used in maintenance, but now it is not used

Egor Ivkov, [02.04.21 18:44]
[In reply to Ivan Rybin]
Ok